spec/rails_helper.rb

```diff
@@ -71,6 +71,7 @@
   Capybara.register_driver :chrome_headless do |app|
     options = ::Selenium::WebDriver::Chrome::Options.new
     options.args << '--headless'
+    options.args << '--window-size=1500,850'
```

Comment:
@maxkadel diagnosed this issue!  The navigation menu spec was failing since the default window size was too small to display the menu entries (it was collapsing to a hamburger menu).